### ISSUE TITLE
problem at 07. Directory of Files Options

### ISSUE THREAD

**Issue by yzila** [mabusaa/argocd-course-issues-tracker#42]
Hi impossible to validate the phase Set recurse option. Application is deployed with de deployment and i can't validate :/

**Comment by christensenjairus** [comment]
I second this

**Comment by MouadOr** [comment]
I've got the same issue

**Comment by mabusaa** [comment]
I tried the scenario, working fine for me, in set recurse step, do you guys sync or keep out of sync ? the step expect you to keep it out of sync and validate.

**Comment by kaymid1** [comment]
> I tried the scenario, working fine for me, in set recurse step, do you guys sync or keep out of sync ? the step expect you to keep it out of sync and validate.

You are right! The lab asked us to first apply application without recurse > Sync > Update with recurse: true > Then just click "Check" (do not Sync in this step).
